Templates/Applications/servicenow.json

```diff
@@ -80,7 +80,7 @@
     ]
   },
   "applicationType": "SAML",
-  "templateName": "SAML",
+  "templateName": "Servicenow",
```

Comment:
is it one word, or should it be "Service Now"?

Reply:
I don't know if there is some specification for these right now, since is just for sparkles I was just changing this so 4799 will be fixed. These templates probably needs work anyway.